src/test/java/com/scriptbasic/testprograms/TestPrograms.java

```diff
@@ -61,6 +61,9 @@ public void testPrograms() throws Exception {
         codeTest("TestEmpty.bas", "");
         codeTest("TestPrintHello.bas", "hello");
         codeTest("TestIf.bas", "111");
+        codeTest("TestSelect1.bas", "111111");
+        codeTest("TestSelect2.bas", "2468");
```

Comment:
I would like to have a test case where

a case is selected and it invokes a subroutine that also contains a `select case`.

Reply:
implemented in TestSelect3 (recursive call)